gulp/tasks/upload-preview.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        gitUtils = require('../util/gitUtils'),
+        childProcess = require('child_process'),
+        previewDomain = 'ux-test.edx.org',
```

Comment:
That's an interesting question. We could make it an environment variable that everyone has to set (since you already have to set AWS environment variables).